/spawn_targets_service:
  ros__parameters:
    object_name: "Kare"
    number_of_target: 1
